src/components/FeedHeader/components/Controls/Controls.scss

```diff
@@ -85,8 +85,9 @@ $block: '.#{$namespace}feed-controls';
                 padding: 4px;
 
                 border: none;
-
-                border-bottom: 1px solid var(--yc-color-line-generic);
+                border-bottom: 1px solid var(--g-color-line-generic);
+                border-bottom-right-radius: 0;
```

Comment:
why?

Reply:
<img width="360" alt="Screenshot 2023-08-03 at 17 29 17" src="https://github.com/gravity-ui/blog-constructor/assets/36075690/6800efad-4d2f-4570-9a07-a6f7a1b815bb">
so that there are no curves here

Reply:
are this border changes from another libraries?

Reply:
the rounding is inherited from the parent component, we configure them